scripts/api.js

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const fs         = require('fs');
+const path       = require('path');
+const yaml       = require('js-yaml');
+const config     = require('../config');
+const sri        = require('./sri.js');
+
+const configFile = config.getConfigPath('_api.yml');
+
+const semver = require('semver');
+
+function getDirectoryNames(source, ignore = []) {
+    return fs.readdirSync(source).map((name) => ignore.indexOf(name) === -1 && name).filter((name) => {
+        if (!name) {
+            return false;
+        }
+        const absolute = path.join(source, name);
+        const stat = fs.lstatSync(absolute);
+
+        // @todo Remove symlink check when twitter-bootstrap files are moved
+        // to the proper directory.
```

Comment:
This cannot happen because of limitation of AWS.

Reply:
Hm. Curious. What exactly is the limitation? I can update the code/commenting accordingly.

Reply:
It doesn't allow symlinks IIRC. Maybe @jmervine has more info on this, since I don't recall the exact details rn.

Reply:
FWIW, this really only matters locally when running the script to generate the new `_api.yml` file. This wouldn't actually be executed on AWS.